docs/pages/foundations/iconography/library.js

```diff
@@ -53,72 +54,79 @@ type IconData = {|
     | 'Time'
     | 'Toggle'
     | 'Utility and tools',
+  description: string,
 |};
 
-function IconTile({ iconName, onTap }: {| iconName: IconName, onTap: () => void |}) {
+function IconTile({
+  iconName,
+  iconDescription = 'Description coming soon',
+  onTap,
+}: {|
+  iconName: IconName,
+  iconDescription: string,
+  onTap: () => void,
+|}) {
   const [hovered, setHovered] = useState();
 
   return (
-    <TapArea
-      rounding={2}
-      tapStyle="compress"
-      onTap={onTap}
-      onMouseEnter={() => setHovered(true)}
-      onMouseLeave={() => setHovered(false)}
-      onFocus={() => setHovered(true)}
-      onBlur={() => setHovered(false)}
-    >
-      <Box
-        borderStyle="sm"
+    <Tooltip text={iconDescription} accessibilityLabel={iconDescription} idealDirection="down">
```

Comment:
Should the a11yLabel just be "" if the tooltip text is the same?

Reply:
The tooltip text only gets read on focus with keyboard if it's in the a11y label (otherwise folks would have to quick nav it, so I think this is a better experience)